client/src/App.jsx

```diff
@@ -35,7 +35,6 @@ import EditJobLead from "./pages/edit-job-lead";
 // helper functions
```

Comment:
Be sure to match the state of app.jsx on your branch with the state of it on main